shared/main-shared.native.js

```diff
@@ -85,34 +96,31 @@ class Main extends Component<void, any, void> {
   }
 }
 
-// $FlowIssue
-const connector = connect(
-  ({
```

Comment:
and if aspects of the store are nullable its a mess and it won't work with immutable anyways so i think long term it'll disappear